### PR TITLE
Alpha beta , needs work

### PR DESCRIPTION
**Pull request by Rmunitz-oss**

### CODE REVIEW

**Comment by Rmunitz-oss** [comment]
Main issues:
Need an enum to take care of if player is MAX or MIN (according to current logic, user is MIN and computer is MAX)
Board class needs to keep track of currentPlayer, many methods need that info (which color) , and alpha beta needs player (max vs min) passed as a param. 
Maybe a Player object should be created to contain both pieces of info? (Color and max or min)

File: CheckersGame/Board.cs [hunk 1]
```diff
@@ -48,11 +48,74 @@ public void SetUpBoard(PieceColor computerColor, PieceColor playerColor)
...
+        public double GetHeuristicValue()
+        {
+            /*
+             * TO DO
+             * evaluate current board
+             * (number of pieces left, weight kings>pawns, number of legal moves left
+             */
+            return 0.0;
+        }
+        //do we have a move class? or just acess through x,y coordinates?
```
Comment (scroitoru) [inline CheckersGame/Board.cs:79]: there is a Move class, but not a getLegalMoves()

Comment (Rmunitz-oss) [reply CheckersGame/Board.cs:79]: should be a board method, based on current board and player, that returns a list of possible legal moves

Comment (Rmunitz-oss) [reply CheckersGame/Board.cs:79]: as in, this needs to be written still

**Pull request merged**